azure-pipelines.yml

```diff
@@ -0,0 +1,209 @@
+    displayName: Component Detection
+
+  - publish: $(Build.StagingDirectory)\drop
+    displayName: Publish drop
+    artifact: drop
+    enabled: false
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-fxcop.FxCop@2
+    displayName: 'Run FxCop'
+    inputs:
+      inputType: Basic
+      targets: 'bin\$(BuildConfiguration)\ApiPort*\net4*\**\ApiPort*.*;bin\$(BuildConfiguration)\ApiPort.Offline\net4*\**\Microsoft.Fx.*.dll;'
+      recursive: false
+      directory: 'C:\Windows\Microsoft.NET\Framework\v4.0.30319'
+      verbose: true
+    enabled: false
+    continueOnError: true
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@3
+    displayName: 'Run BinSkim '
+    inputs:
+      InputType: Basic
+      AnalyzeTarget: '$(Build.StagingDirectory)\drop\Microsoft.Fx.*.dll;$(Build.StagingDirectory)\drop\ApiPor*.dll;$(Build.StagingDirectory)\drop\ApiPor*.exe'
+      AnalyzeSymPath: $(Build.StagingDirectory)\symbols
+    continueOnError: true
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@1
+    displayName: 'Run APIScan net461'
+    inputs:
+      softwareFolder: '$(Build.StagingDirectory)\drop\raw\binaries\net461'
+      softwareName: 'API Port'
+      softwareVersionNum: 4.XX
+      symbolsFolder: '$(Build.StagingDirectory)\drop\raw\symbols\net461'
+      isLargeApp: false
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
+    displayName: 'Publish Security Analysis Logs'
+    condition: succeededOrFailed()
+
+- job: Linux
+  pool:
+    vmImage: Ubuntu 18.04
+  variables:
+    BuildNetCoreAppTarget: netcoreapp2.1
+    RestoreBuildProjects: src/ApiPort/ApiPort/*.csproj
+  steps:
+  - powershell: .\init.ps1
+    displayName: init.ps1 script
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet restore
+    inputs:
+      command: restore
+      projects: $(RestoreBuildProjects)
+      feedsToUse: config
+      nugetConfigPath: NuGet.config
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet build
+    inputs:
+      projects: $(RestoreBuildProjects)
+      arguments: '--configuration $(BuildConfiguration) -f $(BuildNetCoreAppTarget) --no-restore'
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet test
+    inputs:
+      command: test
+      projects: |
+        tests/**/*[Tt]ests/*.csproj
+        !tests/**/*/ApiPortVS.Tests.csproj
+      arguments: '--configuration $(BuildConfiguration) --framework $(BuildNetCoreAppTarget) --no-build'
+
+  - task: DotNetCoreCLI@2
+    displayName: dotnet publish
+    inputs:
+      command: publish
+      publishWebProjects: false
+      projects: $(RestoreBuildProjects)
+      arguments: '--configuration $(BuildConfiguration) -f $(BuildNetCoreAppTarget) --output $(build.artifactstagingdirectory) --no-build'
+      zipAfterPublish: True
+
+  - publish: $(build.artifactstagingdirectory)
```

Comment:
previous UI pipeline definition has artifact name as "drop". Should we define it too here, or it is default value anyway?

Reply:
The default value is "Linux", I guess named after the job name.
The original linux pipeline used `drop` as an artifact name. 

We could name this one `drop-Linux` and the Windows one `drop-Windows` if you'd like.

Reply:
I've renamed the drops as I proposed.